src/components/HomeFooter.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+const HomeFooter = () => (
+  <div>
+    <div className="text-md-left text-center p-4 bg-dark text-light">
+      <div className="container">
+        <div className="row">
+          <div className="my-3 col-lg-4 col-md-6">
+            <h4>Authors Haven</h4>
+            <p className="text-muted">July 25 2018</p>
+            <p className="my-3 text-muted">
+              St. Catherines, Off Thika Road, Nairobi
+              <br />
+            </p>
+          </div>
+          <div className="my-3 col-lg-4 col-md-6">
+            <h5 contentEditable="true">Ask us a question</h5>
+            <form>
+              <fieldset className="form-group my-3">
+                <input type="email" className="form-control" id="Input Email" placeholder="Email" />
+              </fieldset>
+              <fieldset className="form-group my-3">
+                <input
+                  type="message"
+                  className="form-control"
+                  id="Input Message"
+                  placeholder="Message"
+                />
+              </fieldset>
+              <button type="submit" className="btn btn-outline-primary">Submit</button>
+            </form>
+          </div>
+          <div className="col-lg-1" />
+          <div className="my-3 col-lg-3">
+            <h5>Follow</h5>
+            <a href="https://www.facebook.com" target="blank">
+              <i className="fa fa-facebook-square text-muted fa-3x m-2" />
```

Comment:
Some of the repeating elements could be rendered by a different component using a loop to keep the code DRY.